src/reducers/entities/posts.js

```diff
@@ -145,13 +148,14 @@ function handleReceivedPosts(posts = {}, postsInChannel = {}, postsInThread = {}
 
     // Sort to ensure that the most recent posts are first, with pending
     // and failed posts first
-    postsForChannel.sort((a, b) => {
+    const filteredPostsForChannel = postsForChannel.filter((id) => nextPosts[id]).sort((a, b) => {
```

Comment:
Why would we need to filter out post ids for posts that don't exist? Is there something that normally causes that to happen, or are you just being extra safe here?

Reply:
I'm seeing `undefined` after deleting combined post; somehow the ID of combined post remained in the postsForChannel.  Not sure what I'm missing there but that's something I need to investigate further.

Reply:
Fixed this by adding `channel_id` to combined post.